lib/hawkular/metrics/metric_api.rb

```diff
@@ -191,19 +203,15 @@ def raw_data(ids, starts: nil, ends: nil, limit: nil, order: nil)
       # @param bucketDuration [String] optional interval (default no aggregation)
       # @return [Array[Hash]] datapoints
       # @see #push_data #push_data for datapoint detail
-      def get_data_by_tags(tags, starts: nil, ends: nil, bucketDuration: nil)
-        params = { tags: tags_param(tags), start: starts,
-                   end: ends, bucketDuration: bucketDuration }
+      def get_data_by_tags(tags, starts: nil, ends: nil, bucketDuration: nil, buckets:nil)
```

Comment:
Also, maybe (in a separate-PR) we should deprecate this `get_data_by_tags` and call it just `data_by_tags`... (using an alias).